src/components/CommentsList.tsx

```diff
@@ -0,0 +1,47 @@
+import { Comment } from '../types/Comment';
+
+type CommentsListProps = {
+  comments: Comment[];
+  handleDeleteComment: (url: string, msg: string, id: number) => void;
+};
+
+export const CommentsList = ({
+  comments,
+  handleDeleteComment,
+}: CommentsListProps) => {
+  const handleClick = handleDeleteComment.bind(
```

Comment:
Are you sure you need this bind call here? 'this' should not be needed for functional components.

Reply:
I wanted to give this function some arguments before click event happen.